src/software/amazon/ion/IntegerType.java

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2016 Amazon.com, Inc.  All rights reserved.
+
+package software.amazon.ion;
+
+/**
+ * Indicates the Java type of an Ion <code>int</code> value.
```

Comment:
I suggest our preferred style should be to use the more succinct JavaDoc syntax `{@code ...}` rather than `<code>...</code>`.